algorithms/CPlusPlus/Graphs/kruskal-algorithm.cpp

```diff
@@ -129,6 +129,14 @@ void KruskalMST(Graph* graph)
 		<< endl;
 }
 
+// Helper function which takes in src, dest, weight, index, address of graph as an argument 
+// to update the value of graph for respective index
+void helperFunc(int s, int d, int w, int idx, Graph** graph){
```

Comment:
I think the function should be renamed to something which tells more about the what the function does and be more helpful. ( like `updateGraph ()` or something along those lines ).

Reply:
Yeah, I agree, I will update the function name.

Reply:
@Arsenic-ATG I have updated the PR.